app/assets/javascripts/SourceCodeGlower/AnnotationTextDisplayer.js

```diff
@@ -72,6 +72,7 @@ AnnotationTextDisplayer.prototype.hide = function() {
 // Show the displayer
 AnnotationTextDisplayer.prototype.show = function() {
   this.display_node.style.display = 'block';
+  reloadDOM();
```

Comment:
I'm still new to the javascript world. If it's not defined, how come it successfully gets called?